Prox/Prox/Models/Event.swift

```diff
@@ -13,6 +14,48 @@ class Event {
     var endTime: Date?
     var url: String
 
+    private static var numberOfEventNotificationStrings: Int = {
+        let key = RemoteConfigKeys.numberOfEventNotificationStrings
+        return FIRRemoteConfig.remoteConfig()[key].numberValue!.intValue
+    }()
+
+    private static var eventNotificationStrings: [String] = {
+        var eventNotificationStrings = [String]()
+        for i in 1...Event.numberOfEventNotificationStrings {
+            let key = RemoteConfigKeys.eventNotificationStringRoot + "\(i)"
+            if let string = FIRRemoteConfig.remoteConfig()[key].stringValue {
+                eventNotificationStrings.append(string)
+            }
+        }
+        return eventNotificationStrings
+    }()
+
+    var notificationString: String {
+        let randomIndex = Int(arc4random_uniform(UInt32(Event.numberOfEventNotificationStrings)))
+        return Event.eventNotificationStrings[randomIndex]
+    }
+
+    private static var numberOfPlaceDisplayStrings: Int = {
+        let key = RemoteConfigKeys.numberOfPlaceDetailsEventStrings
```

Comment:
nit: it'd be nice to put the duplicated code in a helper fn, but to save time, wfm!